lib/cmake_modules/config.cmake.in

```diff
@@ -0,0 +1,3 @@
+include(CMakeFindDependencyMacro)
+
+include("${CMAKE_CURRENT_LIST_DIR}/@NGHTTP3_TARGETS_EXPORT_NAME@.cmake")
```

Comment:
A newline is missing at he end of this file.

What do you think of moving this file to the existing `cmake` directory instead?
Along with the generated `.cmake` modules above.

Reply:
It's okay for me